### PR TITLE
DBZ-7373 Zookeeper container not being deleted quick fix + added wait…

### PR DESCRIPTION
**Pull request by novotnyJiri**
https://issues.redhat.com/browse/DBZ-7373

### CODE REVIEW

File: debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/DockerKafkaController.java [hunk 1]
```diff
@@ -51,7 +57,8 @@ public String getTlsBootstrapAddress() {
     @Override
     public boolean undeploy() {
         container.stop();
-        return container.isRunning();
+        zookeeperContainer.stop();
+        return zookeeperContainer.isRunning();
```
Comment (jpechane) [inline debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/DockerKafkaController.java:61]:
```suggestion
        return container.isRunning() && zookeeperContainer.isRunning();
```
but I am a bit confused on te return value, it seems to me accroding to JavaDoc of the method on interface it should return negated value of `isRunning()` as I suppos that successful `undeploy` means that container is NOT running. If it is the case it should be fixed at other controllers too.

**Review by jpechane — CHANGES_REQUESTED** [review]
@novotnyJiri Please see the comment about `undeploy` method

File: debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/DockerKafkaController.java (hunk 1 above)
Comment (novotnyJiri) [reply debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/DockerKafkaController.java:61]: Great catch with the JavaDoc discrepancy! I will fix all the controllers accordingly.

**Comment by jpechane** [comment]
@novotnyJiri Applied, thanks!

**Pull request merged**